Short-Answer/Algorithms_Answers.md

```diff
@@ -2,14 +2,41 @@
 
 ## Exercise I
 
-a)
+a) n is our input. If n gets bigger, this program will perform more operations (for loops). Since n is cubed on line 2, as our input gets bigger, the number of operations has a polynomial growth at a worst case scenario of O(n^3). 
```

Comment:
Just reiterating what we talked about in 1:1, since the loop is adding n squared each time through on its way to n cubed, the effective time complexity is O(n)